score.py

```diff
@@ -43,6 +44,8 @@ def score(self) -> None:
         true_negatives = 0
         false_positives = 0
 
+        fmt = "{0:." + str(self.precision) + "f}%"
```

Comment:
Can you change this too `percentage = f"{0:.{self.precision}f}%"`

Reply:
I tried but failed.  f strings are a little unfamiliar still.